prow/cmd/deck/static/tide/tide.ts

```diff
@@ -322,7 +335,12 @@ function addBlockersToElem(elem: HTMLElement, pool: TidePool): void {
     }
 }
 
+let tooltipCounter = 0;
 function addToolTipToElem(elem: HTMLElement, tipElem: Node): void {
+    if (!elem.id) {
```

Comment:
I don't really like that adds an ID - that seems like it could be surprising. It might be preferable to instead add the ID elsewhere, and document that this accepts an ID (or, better, change the API so it actually _takes_ an ID instead of a node)